atomix/cluster/src/main/java/io/atomix/raft/partition/RaftPartitionGroupConfig.java

```diff
@@ -37,6 +37,8 @@
   private Duration electionTimeout = DEFAULT_ELECTION_TIMEOUT;
   private Duration heartbeatInterval = DEFAULT_HEARTBEAT_INTERVAL;
   private RaftStorageConfig storageConfig = new RaftStorageConfig();
+  private int maxAppendsPerFollower = 2;
```

Comment:
iirc, these will get serialized by kryo right? just to be safe, this should be fine from now on with the new compatible serializer, but did we test this? i guess it only breaks rolling upgrade, which is broken now anyway though :thinking:

Reply:
Tbh I always worry to touch the config and this is so stupid :sweat_smile: I can't tell what happens currently

Reply:
@npepinpe do I need to do something here?

Reply:
No it was just a question - I think it would be fine with the PR Miguel will merge with the new version serializer, where you can add new fields. @MiguelPires ?